### ISSUE TITLE
Does it work in medical dataset?

### ISSUE THREAD

**Issue by joker616** [MIVRC/MLEFGN-PyTorch#4]
Hello ,my research interest is X-ray imaging and its applications.I want to use your code CT projection denoising because the excellent results in your paper.But what confusing me is that the data value fluctuate around 1.just like 1.000058 or 0.999954.
At the first,i tried tranforms my data to .png,it does works，resuls is pretty good.But the value is between 0-255,it can't reconstructed successfully because it losses many information.And I tried modify the code to input a matrix output a matrix with my true data value without normalized.But the L1 loss is too small and soon to zero.

Can you tell me how can I solve this issue?Like use normalization before the training or modify the loss function.Or if you are intrested in my area we can incooperate to do more thing.Like we can provide simulation and experimental dataset include attenution-based \phase contrast CT and expert knowledge. you can give us some advice about machine learning. 
                                                                                                                                                                        Best regards.
